grakn-client/src/main/java/ai/grakn/client/Client.java

```diff
@@ -55,7 +57,20 @@
             return reader.lines().collect(joining("\n"));
         }
     };
-    
+
+    public static void main(String[] args) throws IOException {
+        String confPath = System.getProperty("grakn.conf");
+        Properties properties = new Properties();
+        try (FileInputStream stream = new FileInputStream(confPath)) {
```

Comment:
I think here we should check if the file exists and print a nice, helpful message if it does not.